packages/pds/src/services/repo/index.ts

```diff
@@ -165,4 +169,100 @@ export class RepoService {
       }),
     )
   }
+
+  async report(info: {
```

Comment:
similarly, these feels like a separate service from the RepoService which is mainly around repo mutations

Reply:
I was following the same line of thinking as in my last comment, and totally down to move some things around.  What would you say to a `ModerationService`, where reports and the views of actions/reports could live?  The server admin actions of taking a moderation action and resolving a report could stay on the `AdminService`, or we could absorb them into the `ModerationService`.

It's a bit of a balance between colocating moderation functionality in a single service versus keeping admin and non-admin functionality separate from each other.  I usually tend towards the former, and treat the service/application layer as being privileged/non-authorization-aware, but there are tons of different reasonable ways to do it!

Reply:
Yup `ModerationService` sounds great 👍 
I don't have a strong opinion on it co-locating the admin actions in there as well, altho my instinct is probably to do so 🤔